ops/dev/vagrant/dev_appserver.sh

```diff
@@ -1,21 +1,9 @@
 #! /bin/bash
 set -e
 
-function get_config_prop {
-    local prop_name="$1"
-    local dev_config_file=""
-    if [ -f "tba_dev_config.local.json" ]; then
-        local dev_config_file="tba_dev_config.local.json"
-    fi
-
-    jq -c -r -s ".[0] * (.[1] // {}) | .$prop_name | select (.!=null)" tba_dev_config.json $dev_config_file
-}
-
-function get_project_from_key {
-    local key_path="$1"
-    jq -r '.project_id' "$key_path"
-}
+source ops/dev/vagrant/config.sh
 
+auth_use_emulator=$(get_config_prop auth_use_emulator)
```

Comment:
- [ ] Let's default this to true if it doesn't exist. Or better - maybe we have this be a `auth_use_prod` or something to make it less weird.